tests/phpunit/tests/comment/query.php

```diff
@@ -2471,7 +2471,7 @@ public function test_search() {
 			)
 		);
 
-		$this->assertSameSets( array( $c1, $c2, $c3, $c4, $c5 ), $found );
+		$this->assertSameSets( array( $c1, $c3, $c4, $c5 ), $found );
```

Comment:
Let's add tests here for a privileged users being able to query by this field.

Reply:
d9d601a28efef2026f9df8d752fff6137d70bff9